pipelines/managed/fbc-release/README.md

```diff
@@ -20,6 +20,10 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                                      | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                             | No       | -                                                         |
 
+## Changes in 4.5.0
+* Add `prepare-fbc-release` task to the Pipeline, which adds new information required by `add-fbc-contribution` to
```

Comment:
So probably `add-fbc-contribution` should have `runAfter: prepare-fbc-release` in the pipeline, no?

Reply:
It does not need because the `prepare-fbc-release` was set to `runAfter: update-ocp-tag` that is in the list of `runAfter:` of `add-contribution-to-index-image`.

Reply:
I don't think that's enough:
- `prepare-fbc-release` has `runAfter: update-ocp-tag`
- `add-fbc-contribution-to-index-image` has `runAfter: update-ocp-tag`

But we need to make sure `add-fbc-contribution-to-index-image` runs only after `prepare-fbc-release` - the two above will not guarantee it, no?

Reply:
added!